components/bl-webcamPhoto-component/src/helpers/file.js

```diff
@@ -0,0 +1,12 @@
+export async function dataURLToBlob(dataURL) {
+  const response = await fetch(dataURL);
```

Comment:
wouldn't it better to use sync code?

https://stackoverflow.com/questions/23150333/html5-javascript-dataurl-to-blob-blob-to-dataurl/30407959#30407959